### PR TITLE
Removal depreceated functions and Addition of Installation Guide

### PR DESCRIPTION
**Pull request by nayakastha**

### CODE REVIEW

**Review by abhibhaw — APPROVED** [review]
I hope it's building properly now😂

File: README.md [hunk 1]
```diff
@@ -4,14 +4,52 @@
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
 # About App
-- The project repository of the mobile app development for Institute Counselling Services, NIT Rourkela.
 
+- The project repository of the mobile application for the Institute Counselling Services, NIT Rourkela. We are using the [Flutter](https://flutter.dev/) framework for the development of the application. This application has more than 3000 active users and is the most popular application at National Institute of Technology Rourkela.
```
Comment (DesignrKnight) [inline README.md:8]: Add a link here to institute website and ICS website

Comment (DesignrKnight) [reply README.md:8]: Also Play Store link

File: scp/lib/timetable/theorySection.dart [hunk 2]
```diff
@@ -179,6 +179,7 @@ Widget _buildCarousel(_TheorySectionState timetableState, BuildContext context,
                 page = pageController.page;
               });
             }
+            return false;
```
Comment (SriramPatibanda) [inline scp/lib/timetable/theorySection.dart:182]: @nayakastha Why this addition?

Comment (nayakastha) [reply scp/lib/timetable/theorySection.dart:182]: `onNotification` expects a bool and previously the function was returning a void. So it didn't make sense

**Review by SriramPatibanda — APPROVED** [review]

**Pull request merged**